migrations/1668360468072_add-trigger-function-to-likesCount-on-posts.js

```diff
@@ -0,0 +1,45 @@
+
+exports.shorthands = undefined
+
+exports.up = (pgm) => {
+
+	pgm.createFunction("computeLikes", [], {
+		returns :"trigger",
+		language : "plpgsql"
+	},
+	`DECLARE postLike post_likes%ROWTYPE;
+
+    BEGIN
+    
+     IF (TG_OP = 'DELETE') THEN
+     postLike = OLD;
+    
+    
+     ELSIF (TG_OP = 'INSERT') THEN
+     postLike = NEW;
+    END IF;
+    
+    UPDATE posts
+    SET "likesCount" = (
+        SELECT COUNT(*) 
+        FROM "post_likes"
+        WHERE "postId" = postLike."postId"
+    )
+    WHERE id = postLike."postId";
+    
+    RETURN postLike;
+    
+    
+    
+    END;`
```

Comment:
lets indent a bit to make the flow a little more readable, like the code in the  if statements and so on
```suggestion
`
   DECLARE postLike post_likes%ROWTYPE;
    BEGIN
    
      IF (TG_OP = 'DELETE') THEN
        postLike = OLD;
      ELSIF (TG_OP = 'INSERT') THEN
        postLike = NEW;
    END IF;

    UPDATE posts
    SET "likesCount" = (
        SELECT COUNT(*) 
        FROM "post_likes"
        WHERE "postId" = postLike."postId"
    )
    WHERE id = postLike."postId";
    
    RETURN postLike;
    
    END;
`
```

Reply:
i think it had a slight lint issue so kindly pull it locally and fix it

Reply:
sure

Reply:
`1668360468072_add-trigger-function-to-likesCount-on-posts.js` should be `1668360468072_add-trigger-function-to-compute-likesCount-on-posts.js`

Reply:
done

Reply:
1668360468072_add-trigger-function-to-likesCount-on-posts.js should be 1668360468072_add-trigger-function-to-compute-likesCount-on-posts.js